engwales-opinion.sty

```diff
@@ -0,0 +1,52 @@
+\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+
+\RequirePackage[explicit]{titlesec}
+\RequirePackage{enumitem}
+
+
+%% Formatting the main title
+\renewcommand\maketitle{%
+  \vspace*{-1.5cm}
+  \begin{center}
+    \bfseries
+    \renewcommand\arrayrulewidth{1.1pt}
+    \begin{tabular}{@{\hspace{12pt}}c@{\hspace{12pt}}}
+      \hline\\[-12pt]
+      \MakeUppercase{\@title}\\\ \\[-12pt]
+      \hline
+    \end{tabular}
```

Comment:
i'm not particularly familiar with package creation. but is there a way to extract this to a shared lib? worst case we can add a tramline.sty (or the like).

Reply:
There is, we just need to create a .sty such as `engwales-tools.sty` and keep all shared utilities in there.

Reply:
Though this should probably be handled in it's own branch